docker/driver.go

```diff
@@ -0,0 +1,96 @@
+package docker
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+
+	"github.com/bblfsh/performance"
+
+	errors "gopkg.in/src-d/go-errors.v1"
+	"gopkg.in/src-d/go-log.v1"
+)
+
+// One day it will become a config file but now it's to annoying to ship it
```

Comment:
```
// TODO(@lwsanty): Convert this to a configuration file and package it with the binary.
```

Putting your name in doesn't mean you have to actually do it, that just tells us who to ask about it later. 😁
Alternatively, you can link to an issue `TODO(#xxx):`.

Reply:
An alternative approach (which might make it easier to hook in a file later):

```go
func driverName(lang string) (string, error) { return lang + "-driver", nil }
```

Right now it means the same thing, but later we could have this read the aforementioned configuration file and report an error for languages that aren't known. This would also let us get rid of the separate lookup/validation step (see comment at line 32).

Reply:
Another option: There are a bunch of different libraries for compiling static files into a binary ([example](https://godoc.org/github.com/creachadair/staticfile)), we could do that here too:

In `driver.go`, say:

```go
//go:generate go run github.com/creachadair/staticfile/compiledata -pkg driver -out static.go path/to/config.file
```

Then during the build:

```
go generate ./...
go build ./...
```

YMMV, though. (It's fine to leave it for later)

Reply:
Done